<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorPickerPattern</class>
 <widget class="QWidget" name="ColorPickerPattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="SectionContainer" name="sectionProperties">
     <property name="title" stdset="0">
      <string>Properties</string>
     </property>
     <widget class="QWidget" name="widgetProperties">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="2" column="1" colspan="4">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>16</number>
         </property>
         <item>
          <widget class="FormWidget" name="labelRotation">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="label" stdset="0">
            <string>Rotation</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ScrSpinBox" name="numberAngle"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="FormWidget" name="labelSpacing">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="label" stdset="0">
            <string>Spacing</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="ScrSpinBox" name="numberSpacing"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="FormWidget" name="labelFlip">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="label" stdset="0">
            <string>Flip</string>
           </property>
           <property name="font">
            <font>
             <pointsize>8</pointsize>
            </font>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="buttonFlipHorizontal">
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="buttonFlipVertical">
              <property name="text">
               <string/>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="FormWidget" name="labelOffset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="label" stdset="0">
          <string>Offset</string>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ScrSpinBox" name="numberOffsetY"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelX">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ScrSpinBox" name="numberOffsetX"/>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="FormWidget" name="formWidget_8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="label" stdset="0">
          <string/>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="preserveLabelSpace">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelY">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="FormWidget" name="labelSkew">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="label" stdset="0">
          <string>Skew</string>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ScrSpinBox" name="numberSkewY"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="FormWidget" name="labelScale">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="label" stdset="0">
          <string>Scale</string>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ScrSpinBox" name="numberScaleY"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="ScrSpinBox" name="numberScaleX"/>
       </item>
       <item row="0" column="4">
        <widget class="ScrSpinBox" name="numberSkewX"/>
       </item>
       <item row="0" column="3" rowspan="2">
        <widget class="FormWidget" name="formWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="label" stdset="0">
          <string/>
         </property>
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="preserveLabelSpace">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="LinkButton" name="buttonScaleRatio">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="SectionContainer" name="sectionSwatches">
     <property name="title" stdset="0">
      <string>Patterns</string>
     </property>
     <widget class="QListWidget" name="listPattern"/>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="leftMargin">
      <number>8</number>
     </property>
     <property name="topMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>8</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkboxLumAlpha">
       <property name="text">
        <string>Use Luminosity as Alpha</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkboxInvertedPattern">
       <property name="text">
        <string>User inverted Pattern</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkboxFollowsPath">
       <property name="text">
        <string>Pattern follows Path</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormWidget</class>
   <extends>QWidget</extends>
   <header>ui/widgets/form_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SectionContainer</class>
   <extends>QWidget</extends>
   <header>ui/widgets/section_container.h</header>
   <addpagemethod>setWidget</addpagemethod>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>LinkButton</class>
   <extends>QToolButton</extends>
   <header>ui/linkbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>numberOffsetX</tabstop>
  <tabstop>numberOffsetY</tabstop>
  <tabstop>numberScaleX</tabstop>
  <tabstop>numberScaleY</tabstop>
  <tabstop>buttonScaleRatio</tabstop>
  <tabstop>numberSkewX</tabstop>
  <tabstop>numberSkewY</tabstop>
  <tabstop>numberAngle</tabstop>
  <tabstop>numberSpacing</tabstop>
  <tabstop>buttonFlipHorizontal</tabstop>
  <tabstop>buttonFlipVertical</tabstop>
  <tabstop>checkboxLumAlpha</tabstop>
  <tabstop>checkboxInvertedPattern</tabstop>
  <tabstop>checkboxFollowsPath</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
